packages/consumption/src/modules/requests/itemProcessors/deleteAttribute/DeleteAttributeRequestItemProcessor.ts

```diff
@@ -0,0 +1,121 @@
+import { DeleteAttributeAcceptResponseItem, DeleteAttributeRequestItem, RejectResponseItem, Request, ResponseItemResult } from "@nmshd/content";
+import { CoreAddress, CoreDate, CoreErrors as TransportCoreErrors } from "@nmshd/transport";
+import { CoreErrors } from "../../../../consumption/CoreErrors";
+import { DeletionStatus, LocalAttribute, LocalAttributeDeletionInfo } from "../../../attributes";
+import { ValidationResult } from "../../../common/ValidationResult";
+import { GenericRequestItemProcessor } from "../GenericRequestItemProcessor";
+import { LocalRequestInfo } from "../IRequestItemProcessor";
+import { AcceptDeleteAttributeRequestItemParametersJSON } from "./AcceptDeleteAttributeRequestItemParameters";
+
+export class DeleteAttributeRequestItemProcessor extends GenericRequestItemProcessor<DeleteAttributeRequestItem> {
+    public override async canCreateOutgoingRequestItem(requestItem: DeleteAttributeRequestItem, _request: Request, recipient?: CoreAddress): Promise<ValidationResult> {
+        const attribute = await this.consumptionController.attributes.getLocalAttribute(requestItem.attributeId);
+
+        if (!attribute) {
+            return ValidationResult.error(
+                CoreErrors.requests.invalidRequestItem(`The Attribute with the given attributeId '${requestItem.attributeId.toString()}' could not be found.`)
+            );
+        }
+
+        if (!attribute.isOwnSharedAttribute(this.accountController.identity.address)) {
+            return ValidationResult.error(
+                CoreErrors.requests.invalidRequestItem(`The Attribute with the given attributeId '${requestItem.attributeId.toString()}' is not an own shared Attribute.`)
```

Comment:
I'd append some extra info to the error: The attribute ... is not an own shared attribute. **Only your own attributes can be requested to be deleted.** This would give the reader a little more context as to what the problem is. May be something to consider. What do you think?